src/SerilogWeb.Classic/Classic/Enrichers/HttpRequestClientHostIPEnricher.cs

```diff
@@ -69,10 +69,10 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
             if (HttpContext.Current == null)
                 return;
 
-            if (HttpContext.Current.Request == null)
+            if (HttpContextCurrent.Request == null)
```

Comment:
In this instance, using:

```
HttpRequest request;
if (!HttpContextCurrent.TryGetRequest(out request))
    return;
```

Will have the added benefit that the follow-up calls can use the `request` variable and avoid calling the property repeatedly (not a significant performance hit, but still seems cleaner to make the null checks etc. only once).